pkg/utils/requester.go

```diff
@@ -33,16 +33,19 @@ import (
 
 const defaultTTL = 5 * time.Second
 
+// DatafileAPI is used as a default API for retrieving the datafile from the CDN
+const DatafileAPI = "https://cdn.optimizely.com/datafiles"
```

Comment:
This is wrong, we should pass this into the `Get`, `Post`, etc because only the polling manager uses this template. We should look at the requester as just a generic client that can make http requests to whatever URI we give it

Reply:
addressed